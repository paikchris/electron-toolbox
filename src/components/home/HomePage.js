import React from "react";
import os from "os";

export class TempHomePage extends React.Component {
	constructor(props) {
		super(props);

		this.state = { date: new Date() };
	}

	render() {
		return (
			<div>
				<span>{os.release()}</span>
				<h1>Toolboxx</h1>
			</div>
		);
	}
}
