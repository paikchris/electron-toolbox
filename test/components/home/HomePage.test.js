// Link.react.test.js
import React from "react";
import renderer from "react-test-renderer";
import * as Enzyme from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import { TempHomePage } from "./../../../src/components/home/HomePage";

Enzyme.configure({ adapter: new Adapter() });

//SETUP
beforeEach(() => {});
afterEach(() => {});

//EXAMPLE STUFF

//TESTS
describe("Home Page Tests", () => {
	test("TempHomePage Snapshot Test", () => {
		const tree = renderer.create(<TempHomePage />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
