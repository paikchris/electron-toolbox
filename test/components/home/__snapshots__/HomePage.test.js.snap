// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page Tests TempHomePage Snapshot Test 1`] = `
<div>
  <span>
    16.7.0
  </span>
  <h1>
    Toolboxx
  </h1>
</div>
`;
